Update README (#10)
sendbird/quickstart-desk-ios: README.md

### Install Desk SDK for iOS

You can install the Desk SDK through either [CocoaPods](https://cocoapods.org/), [Carthage](https://github.com/Carthage/Carthage) or [Swift Package Manager](https://www.swift.org/package-manager/).

#### - CocoaPods

1. Create or edit your `Podfile`

```bash
# Uncomment the next line to define a global platform for your project
platform :ios, '9.0'

target 'YOUR_PROJECT' do
    # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
    use_frameworks!
    
    # Pods for YOUR_PROJECT
    # Desk
    pod 'SendBirdDesk'
    # UIKit
    pod 'SendBirdUIKit'

end
```
2. Run `pod install`.
3. Open `QuickStart.xcworkspace`.

#### - Carthage

> 1. Add `SendBirdDesk` and `SendBirdSDK` into your `Cartfile` as below:

Reviewer: ```suggestion
1. Add `SendBirdDesk`, `SendBirdSDK` and `SendBirdUIKit` into your `Cartfile` as below:
```

```bash

> github "sendbird/sendbird-desk-ios-framework"
> github "sendbird/sendbird-ios-framework" == 3.0.90

Reviewer: ```suggestion
github "sendbird/sendbird-desk-ios-framework"
github "sendbird/sendbird-ios-framework" == 3.0.90
github "sendbird/sendbird-uikit-ios"
```

```

2. Install the `SendBirdDesk` framework through `Carthage`.

```bash
$ carthage update --use-xcframeworks
```

#### - Swift Package Manager

1. File -> Swift Packages -> Add package dependency...

> 2. Choose Package Repository as the Sendbird UIKit repository with below link:

Reviewer: ```suggestion
2. Choose Package Repository as the SendbirdDesk repository with below link:
```

```bash
https://github.com/sendbird/SendBird-Desk-iOS-Framework.git
```

> 3. Select Up to Next Major rules and click the Next button to add the package.

Reviewer: ```suggestion
3. Select Up to Next Major rules and click the Next button to add the package.

4. Repeat steps 2 and 3 for SendBirdUIKit.
```bash
https://github.com/sendbird/sendbird-uikit-ios.git
```

```
